projects/storefrontlib/cms-components/content/video/video.component.html

```diff
@@ -0,0 +1,25 @@
+<div
+  *ngIf="data$ | async as data; else loading"
+  class="video-container"
+  tabindex="-1"
+>
+  <a *ngIf="data.overlayTitle" [routerLink]="routerLink">{{
+    data.overlayTitle
+  }}</a>
+  <video
+    *ngIf="source"
+    [style.height.px]="getHeight(data)"
+    controls
+    [poster]="thumbnail?.src"
+    [loop]="loop"
+    [autoplay]="autoPlay"
+    [muted]="mute"
+    [attr.aria-label]="'player.label' | cxTranslate"
+  >
```

Comment:
I tested, the poster is gone, and height is not set. where you set the default height `100%`?